.github/CONTRIBUTING.md

```diff
@@ -1,38 +1,37 @@
-Contributing to FOSSProfiles
-============================
+# Contributing to FOSSProfiles
 
 Want to contribute to the FOSSProfiles project? Awesome! Here's some things to
 help you make your contributiong.
 
-
 ## Adding a new profile
 
 First, look at the [YAML example
 profile](https://github.com/FOSSRIT/FOSSProfiles/blob/master/example_profile.yaml)
 to see how the profiles are structured. You can also copy this file and use it
 as a base for making your own profile.
 
-#### Required fields
+### Required fields
 
 All profiles are required to have these fields completed:
 
 * `name`
-* `website`
+* `website` (please include `http[s]://`)
```

Comment:
What is the best way to illustrate this requirement?

Reply:
@kennedy I think this works for now.